sh.c

```diff
@@ -1769,17 +1769,63 @@ void comp_body(ast node) {
 
 // Return if the statement is a break or return statement, meaning that the block should be terminated.
 // A switch conditional block is considered terminated if it ends with a break or return statement.
-bool comp_switch_block_statement(ast node, bool start_in_tail_position) {
+bool comp_switch_block_statement(ast node, bool else_if, bool start_in_tail_position) {
+  ast statement = 0;
+  bool termination_lhs = false;
+  bool termination_rhs = false;
+
   if (get_op(node) == BREAK_KW) {
+    append_glo_decl(wrap_char(':'));
```

Comment:
We only want to add `:` if the rest of the body is empty like it's done in `comp_statement`. Maybe it's easier to add the "is it terminating" analysis to `comp_statement` to reduce duplication?

Reply:
Fixed